transport-classes-io_uring/src/main/java/io/netty/channel/uring/SubmissionQueue.java

```diff
@@ -171,7 +171,9 @@ public String toString() {
     }
 
     long addNop(byte flags, long udata) {
-        return enqueueSqe0(Native.IORING_OP_NOP, flags, (short) 0, 0, 0, 0, 0, 0, udata,
+        // Mimic what liburing does. We want to use a count of 1:
```

Comment:
A count of -1?

Reply:
This was a copy and paste error... Removing the comment